tests/pytests/unit/fileserver/test_s3fs.py

```diff
@@ -1,19 +1,154 @@
+import os
+
+import boto3
 import pytest
+import yaml
+
+# moto must be imported before boto3
+from moto import mock_s3
 
 import salt.fileserver.s3fs as s3fs
+import salt.utils.s3
+from tests.support.mock import patch
+
+
+@pytest.fixture(scope="module")
+def bucket():
+    return "mybucket"
+
+
+@pytest.fixture(scope="module")
+def aws_creds():
+    return {
+        "aws_access_key_id": "testing",
+        "aws_secret_access_key": "testing",
+        "aws_session_token": "testing",
+        "region_name": "us-east-1",
+    }
 
 
-@pytest.fixture
-def configure_loader_modules(tmp_path):
+@pytest.fixture(scope="function")
```

Comment:
```suggestion
@pytest.fixture
```
This was fine as it was, because `function` scope is the default.
Not going to block the PR because of this tough.

Reply:
Yeah, it's not a totally necessary change; I just prefer to be explicit, rather than rely on defaults. This PR is a good example of why 🙂

Up to you though, I don't mind changing it back if you'd like

Reply:
All good